public/js/p3/widget/SubsystemServiceContainer.js

```diff
@@ -97,6 +97,7 @@ define([
 
         this.tabContainer.addChild(this.subsystemsGrid);
       }));
+      /*
```

Comment:
Is this commented out to be used or referred to later? I'd like to get away from the practice of leaving large unused chunks in so that the code is neat and copy/paste errors happen less if this is used later down the line.